### DISCUSSION TITLE
How do I upgrade to the latest version of SmoothWAN on Slate AX

### DISCUSSION THREAD

**Discussion by LunarRed** [SmoothWAN/SmoothWAN#131] [General & Help]
I'm running SmoothWAN on a Slate AX and want to upgrade to the latest version. I can't find any documentation on how to upgrade. Can someone point me in the right direction?

**Comment by TalalMash** [comment]
Hi, just use the WebUI image and visit `System`->`Upgrade`->`Flash image` and uncheck _Keep settings_.
If force upgrade flag is visible, check it.

**Comment by LunarRed** [comment]
I don't have Upgrade under System in the Web UI. Could my version be missing this? 
<img width="967" alt="image" src="https://user-images.githubusercontent.com/3605985/224212030-83feb2aa-05c4-4c01-80f8-bbe3ddae6ae9.png">
<img width="966" alt="image" src="https://user-images.githubusercontent.com/3605985/224212344-09203e31-3bed-49bb-9448-6291d3f66cb8.png">

**Reply by TalalMash** [reply]
My bad it's in `Backup / Flash Firmware`

**Reply by LunarRed** [reply]
Gotcha. Is the file I would upload `SmoothWAN-0.99.9HF4-GL.iNet-Slate-AX-WebUI.tar` or `SmoothWAN-0.99.9HF4-GL.iNet-Slate-AX-Recovery.img`?

Also, does the "Keep settings" checkbox appear after I upload? 
<img width="881" alt="image" src="https://user-images.githubusercontent.com/3605985/224218037-a03b237e-52d2-4f01-a828-9ef5db9e75b1.png">

**Reply by TalalMash** [reply]
Yes, do not keep settings.
"Use the WebUI image and visit System->Upgrade->Flash image and uncheck Keep settings.
If force upgrade flag is visible, check it."
